lib/extensions/polyglot_with_database/coursemology/polyglot/language/python/python2_point7.rb

```diff
@@ -0,0 +1,7 @@
+module Extensions::PolyglotWithDatabase::Coursemology::Polyglot::Language::Python::Python2Point7
```

Comment:
The languages aren't new, these classes are defined in the polyglot gem. I'm just adding the methods needed for the database part to work.

So the polyglot gem defines the stylesheets, javascript, display options; the Coursemology application defines how to interact with the database (the `polyglot_languages` table)